src/main/java/slimeknights/tconstruct/common/config/Config.java

```diff
@@ -329,6 +330,14 @@ public static boolean syncConfig() {
       propOrder.add(prop.getName());
       Util.setTemperaturePref(temperatureCelsius);
 
+      prop = configFile.get(cat, "minFluidHeight", minFluidHeight);
+      prop.setComment("Minimum fluid height to display in smeltery. Make those tiny amounts easier to click on. Can make the Smeltery appear more full than it really is.");
```

Comment:
Should have a bit more info, like that it's in pixels, that it only affects fluids with less amount than required to display properly,..